activiti-api-process-runtime/src/main/java/org/activiti/api/process/runtime/ProcessAdminRuntime.java

```diff
@@ -108,4 +109,10 @@
     void setVariables(SetProcessVariablesPayload setProcessVariablesPayload);
 
     void removeVariables(RemoveProcessVariablesPayload removeProcessVariablesPayload);
+    
+    /**
+     * Get all subprocesses by pages using payload filters
+     */
+    Page<ProcessInstance> subprocesses(GetSubprocessesPayload getSubprocessesPayload,
```

Comment:
Actually this method is not necessary neither you can reuse `processInstances(Pageable pageable,
                                            GetProcessInstancesPayload getProcessInstancesPayload)`